src/Controller/Admin/DescriptionCrudController.php

```diff
@@ -48,9 +49,10 @@ public function configureFields(string $pageName): iterable
             AssociationField::new('Departments')
                 ->formatValue(function ($value, $entity) {
                     $str = $entity->getDepartments()[0];
-                    for ($i = 1; $i < $entity->getDepartments()->count(); $i++) {
-                        $str = $str . ", " . $entity->getDepartments()[$i];
+                    for ($i = 1; $i < $entity->getDepartments()->count(); ++$i) {
+                        $str = $str.', '.$entity->getDepartments()[$i];
```

Comment:
Use array functions (`implode`, `array_map`) and https://www.amitmerchant.com/using-arrow-functions-php/#hello-arrow-functions for stuff like this, e.g.

```php
return implode(', ', array_map(fn (Department $department) => $department->getName());
```